app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
Yeah, I like doing `DescendantLoader.instance.load_subclasses(self)`.
It seems much clearer and is using a public interface.